routes/user.js

```diff
@@ -20,10 +20,11 @@ router.use('/:id/reviews', reviewRouter)
 router.get('/', asyncWrapper(userController.getUsers))
 router.get('/:id', asyncWrapper(userController.getOneUser))
 router.get('/my-profile', setCurrentUserIdAndRole, asyncWrapper(userController.getOneUser))
+router.patch('/me', setCurrentUserIdAndRole, asyncWrapper(userController.updateUserProfile))
 
 router.use(restrictTo(ADMIN))
 
-router.patch('/:id', asyncWrapper(userController.updateUser))
+router.patch('/change-status/:id', asyncWrapper(userController.updateUserStatus))
```

Comment:
why did you add /change-status/? why did you rename to updateUserStatus?

Reply:
i think it can have a place here because this route has only one responsibility for changing status for specific user

Reply:
....for now

it may change any minute + our API is pretty simple so i believe in this case users/:id could be sufficient

@maarekua , what do you think?